### PR TITLE
header

### PR DESCRIPTION
**Pull request by Toshik30**

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            </ul>
+          </nav>
+          <div class="contacts">
+            <a class="number" href="tel:+380953078045">+380953078045</a>
+            <a class="adress"href="https://www.google.com/maps/place/%D0%94%D0%9F%D0%A1+%D0%90%D1%8D%D1%80%D0%BE+%D0%A1%D0%B5%D1%80%D0%B2%D0%B8%D1%81/@50.4478481,30.6182218,17z/data=!4m5!3m4!1s0x40d4c559d0b3e367:0x88d396b689adfc4b!8m2!3d50.4477589!4d30.6202859">ул.Бульвар Верховного Совета 14б</a>
+          </div>
+        </div>
+      </div>
+    </header>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
```
Comment (Vip09vd) [inline app/index.html]: remove it

File: app/scss/style.scss [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+body {
+	font-family: "Open Sans";
+}
+a {
+	text-decoration: none;
+}
+.container {
+	max-width: 1200px;
+	align-items:center;
```
Comment (Vip09vd) [inline app/scss/style.scss]: not work

File: app/scss/style.scss [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  @media (max-width: 600px) { @content; }
+}
+
+body {
+	font-family: "Open Sans";
+}
+a {
+	text-decoration: none;
+}
+.container {
```
Comment (Vip09vd) [inline app/scss/style.scss]: add sizes for tablet and phone

File: app/scss/_fonts.scss [hunk 4]
```diff
@@ -0,0 +1,30 @@
+@font-face {
```
Comment (Vip09vd) [inline app/scss/_fonts.scss]: remove if not used

File: app/scss/style.scss [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+   				margin-left: 5px;
+   				margin-right: 5px;
+   			}
+		}
+	}
+	.contacts {
+		display: flex;
+    	flex-direction: column;
+    	a {
+    		text-decoration: none;
```
Comment (Vip09vd) [inline app/scss/style.scss]: duplicated

File: app/scss/style.scss [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+
+* {
+	box-sizing: border-box;
+}
+li {
+	list-style-type: none;
+}
+header {
+	.container {
+		display: flex;
```
Comment (Vip09vd) [inline app/scss/style.scss]: use %flex-between from _vars

File: app/scss/style.scss [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+		align-items:center;
+		justify-content: space-between;
+	}
+	.logo {
+
+		p {
+			text-transform: uppercase;
+			color: rgba(0,55,255, 0.8);
```
Comment (Vip09vd) [inline app/scss/style.scss]: move colors and font sizes to vars

File: app/scss/style.scss (hunk 3 above)
Comment (Toshik30) [reply app/scss/style.scss]: +

File: app/scss/style.scss (hunk 2 above)
Comment (Toshik30) [reply app/scss/style.scss]: +

File: app/scss/_fonts.scss (hunk 4 above)
Comment (Toshik30) [reply app/scss/_fonts.scss]: use only open sans, open sans bold

File: app/scss/style.scss (hunk 5 above)
Comment (Toshik30) [reply app/scss/style.scss]: +

File: app/scss/style.scss (hunk 6 above)
Comment (Toshik30) [reply app/scss/style.scss]: done

File: app/scss/style.scss (hunk 7 above)
Comment (Toshik30) [reply app/scss/style.scss]: done

**Pull request merged**